app/models/comment.rb

```diff
@@ -2,5 +2,5 @@ class Comment < ApplicationRecord
   validates_presence_of :content
   validates_presence_of :upvotes
 
-  belongs_to :commentable, polymorphic: true 
+  belongs_to :commentable, polymorphic: true
```

Comment:
These polymorphic relationships are a game changer and changes how we can use tables basically forever.